script.js

```diff
@@ -1,4 +1,195 @@
+    userName = input;
+    mode = "game mode";
+    displayOutputValue = `Hello ${userName}, let's play scissors, paper, stone!`;
+  } else if (input == "reverse") {
+    mode = "reverse mode";
+    displayOutputValue = `You have entered an upside down world 🙃`;
+  } else if (mode == "reverse mode") {
+    displayOutputValue = playReverseScissorsPaperStone(input);
+  } else {
+    displayOutputValue = playScissorsPaperStone(input);
+  }
+
+  return displayOutputValue;
 };
+
+// regular game mode
+function playScissorsPaperStone(input) {
+  var programPlays = chooseScissorsPaperStone();
+  // logic of regular scissors paper stone game
+  if (
+    (input == "scissors" && programPlays == "scissors") ||
+    (input == "paper" && programPlays == "paper") ||
+    (input == "stone" && programPlays == "stone")
+  ) {
+    winCount = winCount;
+    lossCount = lossCount;
+    drawCount = drawCount + 1;
+    gameOutcomeOutput = `It's a draw. You have drawed with the computer ${drawCount} time(s) `;
+  } else if (
+    (input == "scissors" && programPlays == "paper") ||
+    (input == "paper" && programPlays == "stone") ||
+    (input == "stone" && programPlays == "scissors")
+  ) {
+    winCount = winCount + 1;
+    lossCount = lossCount;
+    drawCount = drawCount;
+    gameOutcomeOutput = `You win! Congratulations ✨
+    <br> You've won ${winCount} time(s), the computer has won ${lossCount} time(s)`;
+  } else if (
+    (input == "scissors" && programPlays == "stone") ||
+    (input == "paper" && programPlays == "scissors") ||
+    (input == "stone" && programPlays == "paper")
+  ) {
+    winCount = winCount;
+    lossCount = lossCount + 1;
+    drawCount = drawCount;
+    gameOutcomeOutput = `You lose！ Bummer.You've won ${winCount} time(s), the computer has won ${lossCount} time(s)`;
+  }
+
+  // assigning emojis for display purpose
+  var inputEmoji;
+  if (input == "scissors" || input == "reversed scissors") {
+    inputEmoji = "✌️";
+  } else if (input == "paper" || input == "reversed paper") {
+    inputEmoji = "✋";
+  } else if (input == "stone" || input == "reversed stone") {
+    inputEmoji = "✊";
+  }
+
+  var programEmoji;
+  if (programPlays == "scissors") {
+    programEmoji = "✌️";
+  } else if (programPlays == "paper") {
+    programEmoji = "✋";
+  } else if (programPlays == "stone") {
+    programEmoji = "✊";
+  }
+
+  // performance metrics
+  totalNonDrawRounds = winCount + lossCount;
+  if (winCount > lossCount) {
+    var playerPerformance = `So far ${userName}, you've won ${winCount}/${totalNonDrawRounds} rounds that are not draws, keep it up!`;
+  } else if (winCount < lossCount) {
+    playerPerformance = `So far ${userName}, you've won ${winCount}/${totalNonDrawRounds} rounds that are not draws, don't be too disheartened.`;
+  } else if ((winCount = lossCount)) {
+    playerPerformance = `So, ${userName}, you've been winning the same number of times as the computer, see if you can tip the scales in your favour.`;
```

Comment:
Consider abstracting this into a separate function such as checkPerformance as it seems like you repeated this code below. A good tell tale sign of when to create a function is when you repeat the same code more than once